doc/api/deprecations.md

```diff
@@ -2751,6 +2751,18 @@ const moduleParents = Object.values(require.cache)
   .filter((m) => m.children.includes(module));
 ```
 
+<a id="DEP0144"></a>
```

Comment:
@mscdex This comes under DEP0133's title `http connection`, but I was unsure whether to include this under that (since DEP0133's message specifies it's only for `response` and `request`). I say we rename that DEP0133 title to prevent any ambiguity, in case we don't want this under DEP0133.